meson.options

```diff
@@ -15,3 +15,7 @@
 option('enable-debug-checks',
        type: 'boolean', value: false,
        description: 'enable run-time assertions')
+
+option('build-test',
+       type: 'boolean', value: false,
```

Comment:
The option should be named `enable-tests`, and should have a default value of `true`.